### PR TITLE
Update SignalFx docs and example to support events

### PR DESCRIPTION
**Pull request by flands**

### CODE REVIEW

**Review by keitwb — APPROVED** [review]
Great, looks good.

File: exporter/sapmexporter/examples/signalfx-k8s.yaml [hunk 1]
```diff
@@ -85,6 +85,9 @@ data:
           receivers: [signalfx, prometheus]
           processors: [memory_limiter, batch]
           exporters: [signalfx]
+        logs:
+          receivers: [signalfx]
```
Comment (tigrannajaryan) [inline exporter/sapmexporter/examples/signalfx-k8s.yaml:89]: We should probably also add `processors: [memory_limiter, batch]` for uniformness.

File: exporter/signalfxexporter/README.md [hunk 2]
```diff
@@ -68,6 +68,20 @@ exporters:
...
+
+```yaml
+service:
+  pipelines:
+    metrics:
+      receivers: [signalfx]
+      processors: [memory_limiter, batch]
+      exporters: [signalfx]
+    logs:
+      receivers: [signalfx]
```
Comment (tigrannajaryan) [inline exporter/signalfxexporter/README.md:81]: Same here.

File: receiver/signalfxreceiver/README.md [hunk 3]
```diff
@@ -39,4 +44,18 @@ receivers:
...
+
+```yaml
+service:
+  pipelines:
+    metrics:
+      receivers: [signalfx]
+      processors: [memory_limiter, batch]
+      exporters: [signalfx]
+    logs:
+      receivers: [signalfx]
```
Comment (tigrannajaryan) [inline receiver/signalfxreceiver/README.md:59]: Same here.

**Review by tigrannajaryan — APPROVED** [review]
LGTM, just a couple minor comments.

**Pull request merged**